src/phpDebug.ts

```diff
@@ -227,6 +227,7 @@ class PhpDebugSession extends vscode.DebugSession {
         } else {
             serverFileUri = localFileUri;
         }
+        serverFileUri = encodeURI('file://' + serverFileUri);
         return serverFileUri;
```

Comment:
I think I found the actual bug. Instead of this change (and :220), in :224 `localPath` needs to be changed to `localFileUri`. In the end, I pulled url-relative in to resolve URLs, not paths :)
Please check if that is enough to fix the issue.